docs/release-notes/release-2024-04-02.md

```diff
@@ -0,0 +1,118 @@
+---
+title: 0.41.0 March 2024
+---
+
+# Release Notes Version 0.41.0 - March 2024
+
+See TypeSpec Core 0.55 [release notes](https://typespec.io/docs/release-notes/release-2024-04-02)
+
+:::warn
+This release contains breaking deprecations
+:::
+
+## Breaking Changes
+
+### @azure-tools/typespec-client-generator-core
+
+API breaking changes:
+
+- [#451](https://github.com/Azure/typespec-azure/pull/451) adjust generated discriminator property sequence to prevent potential breaking change
+- [#459](https://github.com/Azure/typespec-azure/pull/459) enums are always fixed after we switch to use union to represent extensible enum
+- [#444](https://github.com/Azure/typespec-azure/pull/444) SdkUnionType, SdkEnumType, and SdkModelType will now always have a `.name` property. `.isGeneratedName` is now a boolean that expresses whether the `.name` was generated or described in the tsp
+- [#524](https://github.com/Azure/typespec-azure/pull/524) depcreate getParameterMapping and make .bodyParam on SdkHttpOperation a single optional param instead of list
```

Comment:
```suggestion
- [#524](https://github.com/Azure/typespec-azure/pull/524) deprecate getParameterMapping and make .bodyParam on SdkHttpOperation a single optional param instead of list
```